music21/musicxml/xmlToM21.py

```diff
@@ -4701,9 +4709,13 @@ def xmlToRepeat(self, mxBarline, inputM21=None):
         else:
             raise bar.BarException('cannot handle mx direction format:', mxDirection)
 
-        if mxRepeat.get('times') is not None:
-            # make into a number
-            r.times = int(mxRepeat.get('times'))
+        try:
+            if mxRepeat.get('times') is not None:
+                # make into a number
+                r.times = int(mxRepeat.get('times'))
+        except bar.BarException:
+            # ignore BarException, just let the set of r.times fail silently
+            pass
```

Comment:
I would just wrap the statement that can fail.
```suggestion
        if mxRepeat.get('times') is not None:
            try:
                # make into a number
                r.times = int(mxRepeat.get('times'))
            except bar.BarException:
                # ignore BarException, just let the set of r.times fail silently
                pass
```